File: Bronze_2/1075.js
let fs = require("fs");
let input = fs.readFileSync("./input.txt").toString().split("\n");
const N = BigInt(input[0]);
const F = BigInt(input[1]);

const shortF = input[1];

const numberToString = N + "";
const intNumber = numberToString.slice(0, -2) + "00";
const bigInt = BigInt(intNumber);

const remainder = bigInt % F;
const numberRemainder = Number(F - remainder);
let answer =
  numberRemainder > 10 ? numberRemainder + "" : "0" + numberRemainder;

if (answer == shortF) {
  answer = "00";
}
console.log(answer);

File: Bronze/Bronze_2/1075/1075.cpp
#include <iostream> 
using namespace std;
#include <string>
 
int main()
{
    int N, F;
    cin >> N >> F;

    int quotientOfHundred = N / 100 * 100;
    int remainder = quotientOfHundred % F;
    
    if (remainder == 0) {
        cout << "00" << endl;
        return 0;
    }

    int answer = F - remainder;
    
    string strAnswer;
    if (answer < 10) {
        strAnswer = "0" + to_string(answer);
    } else {
        strAnswer = to_string(answer);
    }

    cout << strAnswer << endl;

    return 0;
}